setup.py

```diff
@@ -47,7 +47,7 @@ def runtests():
         'django_mesh_test_project',
     ],
     include_package_data=True,
-    install_requires=['Django>=1.7', 'django-model-utils',],
+    install_requires=['Django>=1.7', 'django-model-utils', 'django-settings-context-processor',],
```

Comment:
Could you move this to "tests_require"? It's needed by our tests but won't actually be required to install.